schemas/past_officers.schema.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://ocf.berkeley.edu/schemas/past_officers.schema.json",
+  "type": "object",
+  "properties": {
+    "terms": {
+      "type": "array",
+      "items": {
+        "$ref": "#/$defs/term"
+      }
+    }
+  },
+  "required": [
+    "terms"
+  ],
+  "$defs": {
+    "term": {
+      "type": "object",
+      "required": [
+        "termSemester",
+        "termOfficers"
+      ],
+      "properties": {
+        "termSemester": {
+          "type": "string"
+        },
+        "termOfficers": {
+          "type": "object",
+          "required": [
+            "gms",
+            "sms"
+          ],
+          "properties": {
+            "gms":{
+              "type": "array",
+              "items": 
+                {
+                  "$ref": "#/$defs/pastOfficer"
```

Comment:
This isn't ideal because we have to repeat the same `pastOfficer` object every time someone has a position, which means we'll have to repeat their full name multiple times. Ideally we'd only need to write someone's full name once.

Let's do something similar to staff hours (https://github.com/ocf/etc/blob/master/configs/staff_hours.yaml), where we only specify the username here (i.e. make this a string), and at the bottom of the file have a separate section that maps username to full name.

I'm imagining something like this:

```yaml
semester: Fall 2018
officers:
  gm:
    - handle: asai
      start: 2018-05-12
      end: 2018-11-07
...
```

Then at the bottom of the file,

```yaml
fullNames:
  asai: "Andrew Aikawa"
```